src/lib/auth.test.ts

```diff
@@ -18,31 +18,32 @@ const PORTAL_JWT = {
   offering_id: 992
 };
 
-const GOOD_NONCE = "goodNonce";
-const BAD_NONCE = "badNonce";
+const GOOD_NONCE: string = "goodNonce";
```

Comment:
Sorry, I should have been more specific - you don't need types for this as it is inferred.  But the types on the objects below let the compiler/intellisense find errors when/if those objects interfaces change.